### ISSUE TITLE
Display only arm binaries when on arm device

### ISSUE THREAD

**Issue by tsusanka** [trezor/trezor-user-env#147]
When on m1 let's display only relevant arm binaries. I believe this is already done for emulators, let's do it for Bridge as well, we can also move it to select box. Stemmed from this [discussion](https://github.com/trezor/trezor-user-env/pull/140#discussion_r802754070).

Blocked by https://github.com/trezor/trezor-user-env/pull/140
